packages/perseus/src/widgets.ts

```diff
@@ -141,6 +142,12 @@ export const getWidgetScorer = (name: string): WidgetScorerFunction | null => {
     return widgets[name]?.scorer ?? null;
 };
 
+export const getWidgetPublicOptionsFunction = (
+    name: string,
+): PublicWidgetOptionsFunctions | null => {
+    return widgets[name]?.getPublicOptions ?? null;
```

Comment:
I think it should probably return an identity function if it can't find a splitter:

```
    return widgets[name]?.getPublicOptions ?? i => i;
```

That way at the call site we don't have to check if the function exists:

```
const splitStuff = getWidgetPublicOptionsFunction("expression")(stuffToSplit)
```

Reply:
Added!